nft-book/utils/convertMetas.js

```diff
@@ -0,0 +1,118 @@
+import fs from "fs";
+const basePath = process.cwd();
+
+fs.writeFileSync(`${basePath}/output/json/_metadata.json`, "");
+const writer = fs.createWriteStream(`${basePath}/output/json/_metadata.json`, {
+  flags: "a",
+});
+
+writer.write("[");
+const readDir = `${basePath}/output/json`;
+let fileCount = fs.readdirSync(readDir).length - 2;
```

Comment:
Why -2? if you are counting `.DS_Store`, it does not exists on Linux or Windows, so we cannot hard code it